projects/plugins/jetpack/extensions/blocks/sharing-button/variations.js

```diff
@@ -99,6 +106,9 @@ variations.forEach( variation => {
 		blockAttributes.service === variationAttributes.service;
 } );
 
+export const getIconByService = service =>
+	variations.find( variation => variation.attributes.service === service )?.icon;
```

Comment:
Does this mean we can get rid of the default size 24 prop on all our icons?

Reply:
Hmm, yeah, I can try doing that. Thanks!